infrastructure/modules/backend.nix

```diff
@@ -33,15 +33,45 @@ let
     #!/usr/bin/env bash
     set -ex
 
-    # the migrations directory is copied into the output of backend
-    cd ${catcolabPackages.backend}
-    ${lib.getExe pkgs.sqlx-cli} migrate run
+    # Transition migrations from sqlx to sqlx_migrator
+    # Check if we have run sqlx_migrator before, if we have not, mark all migrations as having been applied
+    # If sqlx_migrator has not been run before then the _sqlx_migrations table will exists, but not the
+    # _sqlx_migrator_migrations table.
+    #
+    # We can get rid of this block after all DBs have been updated with sqlx_migrator, new DBs will never
+    # have have had _sqlx_migrations so there will be no need to transition them. 
```

Comment:
Makes sense! Just flagging this so we can create an issue that will remind us to remove this shim once both prod and next have had their migrations migrated (!).

Reply:
New issue is here: https://github.com/ToposInstitute/CatColab/issues/514